beacon_node/beacon_chain/src/schema_change/migration_schema_v26.rs

```diff
@@ -0,0 +1,91 @@
+use crate::persisted_custody::{PersistedCustody, CUSTODY_DB_KEY};
+use crate::validator_custody::CustodyContextSsz;
+use crate::BeaconChainTypes;
+use ssz::{Decode, Encode};
+use ssz_derive::{Decode, Encode};
+use std::sync::Arc;
+use store::{DBColumn, Error, HotColdDB, KeyValueStoreOp, StoreItem};
+use tracing::info;
+
+#[derive(Debug, Encode, Decode, Clone)]
+pub(crate) struct CustodyContextSszV1 {
```

Comment:
Usually we use the first version where the data appeared to name it, but it doesn't matter too much: https://github.com/sigp/lighthouse/blob/stable/beacon_node/beacon_chain/src/schema_change/README.md#naming-conventions

```suggestion
pub(crate) struct CustodyContextSszV24 {
```

Reply:
Thanks, updated!